src/main/java/redlightBack/stockRecommendation/StockRecommendation.java

```diff
@@ -40,39 +40,25 @@ public class StockRecommendation {
 
     private Double dividendYield;  //연배당수익률
 
-    private LocalDateTime updatedAt;
+    private Double currentToHighRatio;  //저평가율
```

Comment:
값이 `null`일 수 없다면 `null`이 될 수 없는 타입인 double을 사용하는 것이 현대적이에요.
```suggestion
    private double currentToHighRatio;  //저평가율
```

Reply:
Yfinance 크롤링으로 데이터 가져와서 계산하는데 가끔 값을 가져오지 못해서 DB에 null이 들어가는 경우가 있더라구요.